lib/mediastreamtrack.js

```diff
@@ -1,10 +1,10 @@
 /* global MediaStreamTrack */
 'use strict';
 
-if (typeof MediaStreamTrack !== 'undefined') {
```

Comment:
we should all this checks for types required by webRTC isWebRTCSupported and use it here.

Reply:
@makarandp0 Since this module depends on a specific API, its OK to check on only that API.